test/auto_test_almond.js

```diff
@@ -3373,9 +3373,7 @@ function main() {
 
     var delegate = new TestDelegate();
 
-    var sempreUrl;
-    if (process.argv[2] !== undefined && process.argv[2].startsWith('--with-sempre='))
-        sempreUrl = process.argv[2].substr('--with-sempre='.length);
+    const sempreUrl = 'https://almond-dev.stanford.edu/nnparser';
```

Comment:
Maybe it is time to change the name?

Reply:
Probably yeah. Although we don't really have a name for the new parser, do we? It is luinet?

Reply:
I don't know if we are still gonna change the luinet name. I guess we can wait until we settle down on the name.